Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -612,10 +613,12 @@ void RemoteRenderingBackend::terminateWebProcess(ASCIILiteral message)
     }
 }
 
+#if PLATFORM(COCOA)
 bool RemoteRenderingBackend::shouldUseLockdownFontParser() const
 {
-    return m_gpuConnectionToWebProcess->isLockdownSafeFontParserEnabled() && m_gpuConnectionToWebProcess->isLockdownModeEnabled();
+    return m_gpuConnectionToWebProcess->isLockdownSafeFontParserEnabled() && m_gpuConnectionToWebProcess->isLockdownModeEnabled() && PAL::canLoad_CoreText_CTFontManagerCreateMemorySafeFontDescriptorFromData();
```

Comment:
Nit: `PAL::canLoad_CoreText_CTFontManagerCreateMemorySafeFontDescriptorFromData()` probably doesn't change after launch. Maybe this could be captured at launch time and remembered for later. (Not necessary for this patch).